OMCompiler/SimulationRuntime/c/simulation/solver/gbode_nls.c

```diff
@@ -246,7 +246,7 @@ NONLINEAR_SYSTEM_DATA* initRK_NLS_DATA(DATA* data, threadData_t* threadData, DAT
     throwStreamPrint(NULL, "Residual function for NLS type %i not yet implemented.", gbData->type);
   }
 
-  nlsData->initializeStaticNLSData(data, threadData, nlsData, TRUE);
+  nlsData->initializeStaticNLSData(data, threadData, nlsData, TRUE, TRUE);
```

Comment:
Does the new `TRUE` has any effect?
Should it be related to the `initNonlinearPattern` passed to other functions?